### DISCUSSION TITLE
Slovenian language support

### DISCUSSION THREAD

**Discussion by uros76** [OHF-Voice/intents#454] [Language Leaders]
Hi. I am a native Slovenian (si locale) with lots of IT and smart home experience. I am willing to help the HA community as a language leader.

**Comment by synesthesiam** [comment]
Thanks @uros76! Could you coordinate with @anaglic and @joselito11 on their current PRs? Once we get those through, I will add you as language leader :+1:

**Comment by Nightbodom** [comment]
@uros76 are you Slovenian language leader?? If you need some help in translating, etc. I can assist you. But I'm no expert in coding so I need someone to tell me what/how to do...

**Comment by andrejs2** [comment]
Hi @uros76 @synesthesiam and @Nightbodom!

I would like to help with translation as well. Prepared some translations of sentences but I'm not sure how to proceed :)

This is my fork: https://github.com/andrejs2/intents 

EDIT: [branch on my fork ](https://github.com/andrejs2/intents/tree/br_light)is the latest and first sentence for lightsHassTurnOn working...Feel free to provide me with guidance and assistance...
